README.md

```diff
@@ -1 +1,157 @@
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. git
+use the following link to setup `git` if you dont have it already installed on your computer
+<p align="left">(<a href="https://git-scm.com/book/en/v2/Getting-Started-Installing-Git">install git</a>)</p>
+
+2. Ruby
+use the following link to setup `Ruby` if you dont have it already installed on your computer
+<p align="left">(<a href="https://www.ruby-lang.org/en/documentation/installation/">install Ruby</a>)</p>
+
+3. Rails
+use the following link to setup `Rails` if you dont have it already installed on your computer
+<p align="left">(<a href="https://guides.rubyonrails.org/getting_started.html#creating-a-new-rails-project-installing-rails">install Ruby</a>)</p>
+
+4. PostgreSQL
+use the following link to setup `PostgreSQL` if you dont have it already installed on your computer
+<p align="left">(<a href="https://www.tutorialspoint.com/postgresql/postgresql_environment.htm">install PostgreSQL</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Leeoasis/ruby-recipe-app.git
+```
+
+Install dependencies:
+
+```sh
+  cd recipe_app
+  bundle install (install all dependencies)
+
+```
+
+Edit `config/database.yml` with your database connection info.
+
+### Usage
+
+The following command can be used to run the application.
+
+```sh
+  rails s
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **AmenMusingarimi**
+
+- GitHub: [@githubhandle](https://github.com/Amen-Musingarimi)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/atmusingarimi/)
+
+👤 **LeslieGudo**
+
+- GitHub: [@githubhandle](https://github.com/Leeoasis)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/leslie-gudo/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[User profile]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Leeoasis/ruby-recipe-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please don't forget to follow me and give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+we would like to acknowledge and thank [Microverse](https://www.microverse.org/) for providing the resources and the platform for this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly consider fixing the broken MIT license and add your own because it gives express permission for users to reuse code for any purpose.